### PR TITLE
Import fenix quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Import fenix quarantine

### CODE REVIEW

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -109,6 +109,8 @@
     <string name="navbar_cfr_message" moz:removedIn="130" tools:ignore="UnusedResources">This bar tucks away as you scroll down for extra browsing space.</string>
     <!-- Text for the message displayed in the contextual feature recommendation popup promoting the navigation bar. -->
     <string name="navbar_cfr_message_2">On a website, this bar tucks away as you scroll down for extra browsing space.</string>
+    <!-- Text for the message displayed for the popup promoting the long press of navigation in the navigation bar. -->
+    <string name="navbar_navigation_buttons_cfr_message">Tap and hold the arrows to jump between pages in this tab\'s history.</string>
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:113]: @rocketsroger Seems like you landed this string. Could you please update the apostrophe here so it follows mozilla [punctuation guidelines](https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-apostrophes-and-quotation-marks-65) (so, updating to a "curly" apostrophe)? thank you!

**Review by Delphine — CHANGES_REQUESTED** [review]

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rocketsroger) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:113]: Thanks for finding this issue.  I'll work with originator to land the fix and update here.

Comment (timc-mozilla) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:113]: I have updated the string with the proper apostrophe. 
This is the patch that will have the change: [patch](https://phabricator.services.mozilla.com/D217632 )

Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: @gabrielluong I wanted to follow up on the linter that was planned to be implemented as part of the monorepo process and that would have caught this. Could you provide an update on its status, please?

Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: Seems like we're running into these errors more and more: https://github.com/mozilla-l10n/android-l10n/pull/787/files#r1693700118

**Pull request merged**